packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -214,6 +222,35 @@ export default {
             commit(UPDATE_FULFILMENT_ADDRESS, addressDetails);
         },
 
+        /**
+         * Post the order details to the Order Placement API and get the `orderId` from the response.
+         *
+         * @param {Object} context - Vuex context object, this is the standard first parameter for actions
+         * @param {Object} payload - Parameter with the different configurations for the request.
+         */
+        placeOrder: async ({ commit, state }, {
+            url, data, timeout
+        }) => {
+            const authHeader = state.authToken && `Bearer ${state.authToken}`;
+
+            const config = {
```

Comment:
Is this config looks to be the same shape or similar throughout? May be worth extracting this out so that the config can accept some options to cater for the other asynchronous calls so it injects a common object which can be extended for other calls and then passed through to axios. That way the other actions can just reuse or extend that without having to repeat it.

Reply:
If you're ok with this, @kevinrodrigues, I think we can do this as a separate PR, as an improvement. There are a few nuances, though. Some APIs expect `application/json`, some others `application/json;v=1`, some expect `x-je-feature`, some `x-je-tenant`, etc.

Reply:
Yup i saw that. More of an improvement thing later I was thinking 👍